plugins/hydra_submitit_launcher/setup.py

```diff
@@ -6,7 +6,7 @@
     LONG_DESC = fh.read()
     setup(
         name="hydra-submitit-launcher",
-        version="1.0.2",
+        version="1.2.0",
```

Comment:
this is released off the master branch?  (what happens to 1.1.0?)

Reply:
1.1.0 as the one released. We generally keep one version higher in setupy. (iirc)

Reply:
yes you are right. sorry I got confused! Didn't realize the release already done and this is a cherry-pick :)

Reply:
Did you release from the master branch?
submitit is probably still compatible, but we should get into the habit of releasing plugins from the appropriate branch.

Reply:
This is released from master. Earlier version was 1.0.2 . I locally bumped the version to 1.1 (before uploading to pypi). Then I bumped the version to 1.2 and made PR against master

Reply:
Hydra in master is diverging from the 1.0 branch and there are already some differences in the examples:

```
$  cat plugins/hydra_submitit_launcher/example/config.yaml 
defaults:
  - override hydra/launcher: submitit_slurm

task: 1
```

In the future please release plugins from the appropriate branch.